reana_server/rest/workflows.py

```diff
@@ -468,6 +470,9 @@ def create_workflow(user):  # noqa
         if is_uuid_v4(workflow_name):
             return jsonify({"message": "Workflow name cannot be a valid UUIDv4."}), 400
 
+        validate_workflow(reana_spec_file, input_parameters={})
+        workspace_root_path = reana_spec_file.get("workspace", {}).get("root_path")
```

Comment:
*note:* validation part is messy. `validate_workflow` also validates workspace_path, which is okay. But, for example, `validate_operational_options,` in addition to validating, is also formatting and returning the output so I cannot remove it.